config.ini

```diff
@@ -0,0 +1,25 @@
+[DEFAULT]
+db_driver = postgresql
+db_user = postgres
+db_pass = postgres
+db_port = 5432
+db_name = gareth
```

Comment:
I would agree, but this is an auto-generated file, so I'll add this as a note on the codegen repo .. will change the default db name to "postgres" unless anyone has another preference.